net/bandwhich/Portfile

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        imsnif bandwhich 0.9.0
+categories          net
+platforms           darwin
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+license             MIT
+
+description         Terminal bandwidth utilization tool (formerly known as \
+                    \"what\")
+
+long_description    bandwhich sniffs a given network interface and records IP \
+                    packet size, cross referencing it with the /proc \
+                    filesystem on linux or lsof on macOS. It is responsive to \
+                    the terminal window size, displaying less info if there \
+                    is no room for it. It will also attempt to resolve ips to \
+                    their host name in the background using reverse DNS on a \
+                    best effort basis.
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  99aade46c602216de3d52a11afcd8b85a1641a3a \
+                    sha256  5ad80f236c748e46dba2b92efc50615c7d76ef421ee8e2d07f983629373c4af2 \
+                    size    1557861
+
+destroot {
+    set gzip [findBinary gzip ${portutil::autoconf::gzip_path}]
+    set manpage "${prefix}/share/man/man1/${name}.1"
+
+    xinstall -m 755 ${worksrcpath}/target/[cargo.rust_platform]/release/${name} ${destroot}${prefix}/bin/
+
+    copy ${worksrcpath}/docs/${name}.1 "${destroot}${manpage}"
+    system "${gzip} -9vf ${destroot}${manpage}"
```

Comment:
MacPorts automatically compresses manpages, doesn't it? This seems unnecessary.

Reply:
You are right.  Removed.